code/modules/mob/living/carbon/human/life.dm

```diff
@@ -847,7 +843,7 @@ var/global/list/brutefireloss_overlays = list("1" = image("icon" = 'icons/mob/sc
 
 	if(species && species.flags & NO_INTORGANS) return
 
-	if(!(species.flags & IS_SYNTHETIC)) handle_trace_chems()
+	handle_trace_chems()

```

Comment:
Shouldn't - previously IPC's didn't process reagents at all, but False added some custom chemicals which means this proc has a use.